final_prep/README.md

```diff
@@ -19,37 +19,41 @@ Some things to consider as you refactor include...
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+ ![Screen Shot 2021-12-18 at 12 53 09 PM](https://user-images.githubusercontent.com/95403666/146654074-ea261d98-a674-4dc4-8373-1959851c5c0a.png)
 - `Add Week 2 Screenshot Here`
+ ![Screen Shot 2021-12-18 at 12 53 16 PM](https://user-images.githubusercontent.com/95403666/146654086-0ca6a82e-0ff5-4616-afcd-965a3e67ae65.png)
 - `Add Week 3 Screenshot Here`
+ ![Screen Shot 2021-12-18 at 12 53 23 PM](https://user-images.githubusercontent.com/95403666/146654090-8ceb9cc1-9366-4962-bce4-f63b8ca6a0d0.png)
+ 
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/CoryBethune/271aa2e3e00afa1b252fc203411d6395
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: https://gist.github.com/CoryBethune/ca18afafa37f666f58973fd0d38970b8
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: https://gist.github.com/CoryBethune/8a49816d9c12cacdcb8535bce087436a 
```

Comment:
👀

`No scheduling constraints. I'll be working on school work once class is out from 5PM to bedtime.`

But what about your 1 hour of no screen time?!?!?!